tmt/steps/prepare/shell.py

```diff
@@ -1,11 +1,14 @@
+from typing import Any, List, Optional
+
 import click
 import fmf
 
 import tmt
 import tmt.utils
+from tmt.steps.provision import Guest
 
 
-class PrepareShell(tmt.steps.prepare.PreparePlugin):
+class PrepareShell(tmt.steps.prepare.PreparePlugin):  # type: ignore[misc]
```

Comment:
is this ignore something we want to remove later right. Should we mark this as `TODO` item? or we will create a task to go through this once we are done @happz ?

Reply:
Yes, add a TODO, please. There is no issue yet, I'll add a note or two to our type annotation issue.

Reply:
```suggestion
# TODO: drop ignore once type annotations between modules enabled
class PrepareShell(tmt.steps.prepare.PreparePlugin):  # type: ignore[misc]
```